edx/xblock-lti-consumer
# [BD-24][TNL-8072]: Improve content presentation and add support for displaying multiple content items

This PR improves the Deep Linking content presentation and adds support for presenting multiple content items in the same XBlock. 

**Testing instructions:**
1. Check out this branch.
2. Set up a local LTI tool that supports Deep Linking (recommended: https://github.com/dmitry-viskov/pylti1.3-django-example) and ngrok tunnel.
3. Create a new block, add the LTI consumer and set up the tool.
4. Go to the LMS and check that the LTI launch is working as expected.
5. Go to Studio and click on the deep linking flow to configure content items.
6. Check that the deep link is configured on Django admin.
7. Go to the LMS and check that the launch works (with the correct content from the deep link). The content should be presented directly in the iframe.
![image](https://user-images.githubusercontent.com/27893385/110809421-a0700080-8263-11eb-9e77-ee2ae59b48d3.png)
8. Go back into django admin and duplicate the Deep Link content for the same block.
![image](https://user-images.githubusercontent.com/27893385/110809577-c8f7fa80-8263-11eb-9380-c98f3c8c4642.png)
9. Reload the LMS page and check that the two content items are presented, and that they launch correctly when clicking on the links.
![image](https://user-images.githubusercontent.com/27893385/110809826-03619780-8264-11eb-8dee-904fec55973f.png)

**IMS conformance testing instructions:**
1. Set up the LTI Reference implementation tool.
2. Do a deep link launch, and select a `ltiResourceLink` content.
3. Go to the LMS and check if the content link selected was launched correctly.
4. Re-do the deep link launch, but this time select more than one content in the IMS tool.
5. Check if the LMS is able to present the supported content types. 

**Reviewers:**
